packages/components/src/internal/components/settings/AuditSettings.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { act } from '@testing-library/react';
+import { AppContext } from '../../AppContext';
+import { getTestAPIWrapper } from '../../APIWrapper';
+import { TEST_FOLDER_CONTAINER, TEST_PROJECT } from '../../containerFixtures';
+import { getFolderTestAPIWrapper } from '../container/FolderAPIWrapper';
+import { AuditSettings } from './AuditSettings';
+import { renderWithAppContext } from '../../test/reactTestLibraryHelpers';
+import { TEST_LKSM_PROFESSIONAL_MODULE_CONTEXT, TEST_LKSM_STARTER_MODULE_CONTEXT } from '../../productFixtures';
+import { TEST_USER_APP_ADMIN } from '../../userFixtures';
+
+describe('AuditSettings', () => {
+    const APP_CONTEXT: Partial<AppContext> = {
+        api: getTestAPIWrapper(jest.fn, {
+            folder: getFolderTestAPIWrapper(jest.fn, {
+                getAuditSettings: jest.fn().mockResolvedValue({ requireUserComments: true }),
+            }),
+        }),
+    };
+
+    test('feature not enabled', async () => {
```

Comment:
any reason to add a test case for "feature enabled but user is not admin"?

Reply:
good thought. Done.